src/index.md

```diff
@@ -1547,7 +1529,63 @@ The Skara tooling includes support for backports. [The official Skara documentat
 [To the top](#){.boxheader}
 :::
 
-# HotSpot development
+# Release Notes
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [release-note label description](#release-note)
+:::
+
+Release notes for a product (e.g. the JDK) are published together with a release of the given product. Release notes describe changes that are important for a user of the product to know about. This is usually things that may affect the user's choice to upgrade to the specific version.
+
+When you write a release note for your feature, be prepared for rather picky review comments about grammar, typos, and wording. This is for the sake of the Java community as a whole, as the language of the release note sets the tone for many blogs and news articles. For a widely used product like the JDK, the release notes are often copied (word by word, including typos) and published to highlight news in the release. This means that we need to take extra care to make sure the text in the release note is correct and has a professional language.
```

Comment:
"When you write a release note" ->. "When writing a release note"


For the general paragraph above,  doesn't the doc team also review the release note and do some minor cleanup?